src/main/kotlin/no/nav/syfo/veileder/VeilederInfo.kt

```diff
@@ -6,4 +6,5 @@ data class VeilederInfo(
     val etternavn: String,
     val epost: String,
     val telefonnummer: String? = null,
+    val enabled: Boolean? = null,
```

Comment:
Trenger kanskje ikke være nullable?

Reply:
Tenkte det var mest riktig for veilederne vi finner i axys, men ikke i MS Graph, siden vi da ikke vet om de er enabled eller ikke.

Reply:
Skjønte ikke helt. Vet vi ikke om de er enabled eller ikke når vi spør axys? 🤔

Reply:
I endepunktet som henter en liste med veiledere for en Nav-enhet så spør vi først axsys om å få alle veiledere på enheten og deretter spør vi msgraph om ekstra info om disse veilederne. Se https://github.com/navikt/syfoveileder/pull/120/files#diff-5ff42d74c7c1b28afd725d663f6db1311e3451f6caea3831a453dd6c4e22b203R29.
Da kan det forekomme at veiledere i axsys ikke finnes i msgraph. 
Inntil vi har migrert bort fra axsys tenker jeg det er greit å fortsatt ha det sånn at endepunktet returnerer disse veilederne, men med `enabled` satt til null siden vi ikke vet.

Reply:
Aha, alle veiledere skal vel finnes i msgraph, så om de finnes i axys så bør vi kanskje bare filtrere de ut? 🤔 
Men om vi beholder det slik så filtrerer vi bare ut de som er enabled=false, og om den er null eller true så viser vi veilederen som vanlig i frontend? 😊

Reply:
Tenkte å vise de uavhengig av flagget i "Søk veileder" i oversikten (slik at de kan gjenfinne personer som er fordelt på veiledere som har sluttet). Men der hvor man skal _tildere_ til veileder så tenker jeg det bare bør være mulig å velge veiledere som har enabled=true.